src/app/essay/create-essay/create-essay.component.ts

```diff
@@ -73,7 +86,20 @@ export class CreateEssayComponent implements OnInit {
             .subscribe(
                 (essay: Essay) => {
                     this.essayService.addEssayElement(essay);
-                    this.onEndSubmission();
+
+                    if(paid) {
+                        let record = {
+                            essayId: essay.id,
+                            value: this.paymentValue * 0.2  // TODO: Check this value.
```

Comment:
could you check and pass me the percentages/amounts paid to the editors for essays for each type of user @Benardi ?